reagurk/public/index.html

```diff
@@ -4,6 +4,15 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+<!-- new meta tags-->
+      <meta property="og:type" content="website">
+      <meta property="og:url" content="<Your website link>">
+      <meta property="og:title" content="Gurkult">
+      <meta property="og:description" content="The Gurkan open source community website.">
+      <meta property="og:image" content="https://i.ibb.co/S0KVsg9/logo.png">
+      <meta property="fb:app_id" content="<Facebook App ID>" />
```

Comment:
Gurkult doesn't have an official Discord (I don't think, so I don't think we need that.